### ISSUE TITLE
Copy clean_cookie and clean_body cleaners from clearpass_api into this repository in the examples

### ISSUE THREAD

**Issue by edthedev** [techservicesillinois/vcrpy-cleaner#35]
## Context

We created cleaners for `clearpass_api` that fully clean an unused cookie, or fully clean an unused response body.

## Tasks

- [ ] create a new file in our example cleaners folder
- [ ] copy in `clean_cookie` from `conftest.py` in `clearpass_api`
- [ ] copy in `clean_body` from `conftest.py` in `clearpass_api`
- [ ] update README with example usage lines of Python (copied from `conftest.py`, including example use of `clean_if`)

**Comment by edthedev** [comment]
Example code for `clean_body`, since we really need #30 before it's useful, so it's not likely to merge into `clearpass_api` today.

```python
@clean_if(uri=f"{URL}/api/endpoint")
def clean_body(request: dict, response: dict):
    response['body'] = '{"status":"success", "data":"cleaned"}'
```

**Comment by edthedev** [comment]
Likely blocked by #30 - practically speaking.

**Comment by edthedev** [comment]
We also have a copy of `clean_cookie` here: https://github.com/techservicesillinois/secops-splunk-domainblock/issues/1
